eng/stages/custom-publish.yml

```diff
@@ -0,0 +1,130 @@
+parameters:
+  dependsOnPublishStages: []
+
+stages:
+# Create one stage per pipeline that needs some extra publish steps.
+- ${{ each dependency in parameters.dependsOnPublishStages }}:
+  - stage: CustomPublish_${{ dependency.dependsOn }}
+    displayName: 'Custom: ${{ dependency.channel.name }}'
+    dependsOn:
+    - ${{ dependency.dependsOn }}
+    variables:
+    - template: /eng/common/templates/post-build/common-variables.yml
+
+    jobs:
+
+    - template: /eng/common/templates/post-build/setup-maestro-vars.yml
+
+    - job: CustomPublish
+      displayName: Custom Publish
+      dependsOn: setupMaestroVars
+      # Only run this job if setup-maestro-vars says the current branch publishes to this channel.
+      # Logic copied from ../common/templates/post-build/channels/netcore-dev-5.yml
+      condition: contains(
+        dependencies.setupMaestroVars.outputs['setReleaseVars.InitialChannels'],
+        format('[{0}]', variables['${{ dependency.channel.bar }}']))
+      pool:
+        name: NetCoreInternal-Pool
```

Comment:
Can the hosted pool be used?

Reply:
Giving that a shot. I believe so, because signing was the absolute reason we couldn't before, and that's in a separate job now.

Reply:
Hosted pool worked! Thanks for the suggestion.